### PR TITLE
Add Gem and rspec tests

### PR DESCRIPTION
**Pull request by damacus**
For some reason rspec/foodcritic is taking a directory but not reading the contents of the directory. 

If we send the Foodcritic constructor a file it behaves correctly, if we send it a folder it ignores it. ¯\_(ツ)_/¯ 

- [ ] Get Rspec to pick up failing rules in directories. 
- [ ] Write tests for all SC foodcritic rules.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,56 @@
+PATH
```
Comment (shortdudey123) [inline Gemfile.lock]: probably shouldn't lock the versions since this will be a gem

File: sous-chefs-foodcritic-rules.gemspec [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+Gem::Specification.new do |spec|
+  spec.name          = 'sous-chefs-foodcritic-rules'
+  spec.version       = '0.0.1'
+  spec.authors       = ['Sous Chefs']
+  spec.email         = ['dan.webb@damacus.io']
+  spec.description   = 'Sous Chefs foodcritic rules'
+  spec.summary       = 'Sous Chefs foodcritic rules'
+  spec.homepage      = 'sous-chefs.org'
```
Comment (shortdudey123) [inline sous-chefs-foodcritic-rules.gemspec]: needs `http://`

File: sous-chefs-foodcritic-rules.gemspec [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  spec.license       = 'Apache'
```
Comment (shortdudey123) [inline sous-chefs-foodcritic-rules.gemspec]: v1? v2?

Comment (damacus) [reply sous-chefs-foodcritic-rules.gemspec]: haven't looked at versioning yet.

Comment (shortdudey123) [reply sous-chefs-foodcritic-rules.gemspec]: i mean which one of these haha
```
Apache-1.0
Apache-1.1
Apache-2.0
```

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+Gemfile.lock
```
Comment (shortdudey123) [inline .gitignore:1]: Need to remove the file too :p

**Pull request closed without merging**